packaging/aznfs/DEBIAN/postinst

```diff
@@ -75,6 +131,14 @@ chmod 4755 /sbin/mount.aznfs
 mkdir -p /opt/microsoft/aznfs/data
 chmod 0755 /opt/microsoft/aznfs/data
 
+if command -v stunnel >/dev/null 2>&1; then
+    INSTALLED_STUNNEL_VERSION=$(stunnel -version 2>&1 | grep -Eo 'stunnel [0-9]+\.[0-9]+' | awk '{print $2}')
+fi
+
+if ! command -v stunnel >/dev/null 2>&1 || (( $(echo "$REQUIRED_STUNNEL_VERSION > $INSTALLED_STUNNEL_VERSION" | bc -l) )); then
+    install_stunnel
+fi
```

Comment:
else log a message saying stunnel version installed is same or newer than required and log both versions